.github/workflows/create-update-dockerfiles-PR.yml

```diff
@@ -2,10 +2,9 @@ name: Create update dockerfiles PR
 
 on:
   workflow_run:
-    workflows: [Update external base images, Update internal base images]
-    types: completed
-  workflow_dispatch:
-
+    workflows: [Update external base images test, Update internal base images test]
+    types:
```

Comment:
Why did we remove the _workflow_dispatch_?

Reply:
needed to revert